picard/ui/mainwindow.py

```diff
@@ -382,6 +382,10 @@ def create_actions(self):
         self.browser_lookup_action.setEnabled(False)
         self.browser_lookup_action.triggered.connect(self.browser_lookup)
 
+        self.albums_search_action = QtGui.QAction(icontheme.lookup('system-search'), _(u"Search similar albums..."), self)
```

Comment:
I realized this sounds weird (affects the track one too). It should be `Search for similar albums...`.